ssn/chapters/System-capabilities.html

```diff
@@ -1,420 +1,290 @@
-            <a href="#SOSASystem">System</a> to a <a href="#SSNSYSTEMSystemProperty">SystemProperty</a> describing the
-            capabilities of the
-            <a href="#SOSASystem">System</a>.
+        <figure class="ssn">
+            <embed title="System properties" alt="System properties" src="images/SSNSystem-property-conditions.svg" style="width: 90%;" />
+            <figcaption>
+                Detailed classes and relationships involved in the description of the conditions (typically an observation collection
+                about the system environment) under which a property (typically about the system capabilities) is valid.
+                <br>
+                In this diagram, unqualified names refer to classes and properties in the <code>sosa:</code> namespace.
+                <br>
+                <em><a href="#Class-diagram">Explanation of the notation used in class diagrams</a></em>.
+            </figcaption>
+        </figure>
+        <p class="note">Beyond the scope of the System Capabilities module, <a href="SOSAProperty">Properties</a> may also be further described with conditions.
+            For example standard meteorological practice defines reference measurement heights, such as 10&nbsp;m for wind and 2&nbsp;m for air temperature.
         </p>
-        <table>
-            <tbody>
-                <tr class="ssn">
-                    <th>Sub property of</th>
-                    <td>
-                        <a>sosa:hasProperty</a>
-                    </td>
-                </tr>
-                <tr>
-                    <th>is Defined By</th>
-                    <td>
-                        <a href="http://www.w3.org/ns/ssn/systems/">http://www.w3.org/ns/ssn/systems/</a>
-                    </td>
-                </tr>
-            </tbody>
-        </table>
-        <div class="buttonpanel">
-            <span class="back">[<a href="#System-capabilities">Back to module overview and examples</a>]</span>
-            <span class="back">[<a href="#Overview">Back to top</a>]</span>
-        </div>
-    </section>
 
-    <section class="specterm sosa" id="SSNSYSTEMSystemProperty">
-        <h5><a href="#SSNSYSTEMSystemProperty">ssn-system:SystemProperty</a></h5>
-        <p class="crossreference">
-            <strong>IRI:</strong> http://www.w3.org/ns/ssn/systems/SystemProperty
+        <p>The following example describes the humidity accuracy property <code>ex:IBSTH2HumidityAccuracy</code>, valid under certain temperature and relative humidity conditions.</p>
+
+        <pre class="example" data-include="./rdf/examples/InkBird-IBS-TH2-properties.ttl" data-include-format="text"></pre>
+    </section>
+    <section>
+        <h4>Guidance on System Capability properties and ranges</h4>
+        <p>The System Capabilities module has been downsized in this edition and its vocabulary extensively reworked with a number of
```

Comment:
We explained that this is heavy revision at the top of this appendix. 
The text in this edition should not dwell on what _used_ to be there, or what we _don't_ use. 
If you really consider it necessary to justify design choices with reference to paths not taken, then this should be in Notes, not the body of the text.

Reply:
True. First I move that to the top of the appendix, we may then discuss whether we want to shrink the design choices description, as it's probably a bit lengthy and not quite as neutral as a spec should be.